headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -1766,6 +1766,191 @@ struct loaded_attack_sprite_data {
 };
 ASSERT_SIZE(struct loaded_attack_sprite_data, 106498);
 
+// Stores the state of the top screen in dungeon mode
+#pragma pack(push, 1)
```

Comment:
See above

Reply:
This one is a little hard for me to say because it's an auto struct from Ghidra and it identified an `int` field in the middle. I don't know if they shouldn't be trusted at all or something, but here it basically means that the packing is required for it to be aligned at the proper position.

Reply:
Are you sure this is correct? Weird packing is possible, but I usually see it as a sign that something is being misunderstood, and should probably be scrutinized further. From what I can tell, `TOP_SCREEN_STATUS` seems to be a double-pointer to some other struct of size 0x230 (it's allocated in the function at 0x22E8874). Usually the access pattern is something like `(*TOP_SCREEN_STATUS) + <offset>`.

Reply:
Yeah okay looks like this is a double pointer indeed, this was one of the first things I did so looks like I made a mistake there.